hw/dv/sv/alert_esc_agent/README.md

```diff
@@ -1 +1,67 @@
-# ALERT Agent
+# ALERT_ESC Agent
+
+ALERT_ESC UVM Agent is extended from DV library agent classes.
+
+## Description
+
+This agent implements both alert(alert_rx, alert_tx) and escalation (esc_rx,
+esc_tx) interface protocols, and can be configured to behave in both host and
+device modes. For design documentation, please refer to [alert_handler
+spec]({{< relref "hw/ip/alert_handler/doc/_index.md" >}}).
+
+### Alert Agent
+
+Alert agent supports both synchronous and asynchronous modes.
+
+#### Alert Device Agent
```

Comment:
looks like it's better to move this section to CIP lib. the agent README doesn't need to include how we use it in CIP.

Reply:
Sure, that sounds good. I will move it to cip lib readme. Thanks Weicai